sdk/spring/azure-spring-boot-starter-keyvault-certificates/README.md

```diff
@@ -316,6 +316,39 @@ spring:
           useInsecureTrustManager: true
 ```
 
+### Refresh certificates when have un trust certificate
+
+When the inbound certificate is not trusted, the KeyVaultKeyStore can fetch 
+certificates from KeyVault if the following property is configured:
+
+```yaml
+azure:
+  keyvault:
+    jca:
+      refresh-certificates-when-have-un-trust-certificate: true
+```
+
+Note: If you set refresh-certificates-when-have-un-trust-certificate=true, your server will be vulnerable
+to attack, because every untrusted certificate will cause your application to send a re-acquire certificate request.
+
+### Refresh certificate periodically
+
+This starter allows you to refresh automatically when the certificate on the KeyVault is modified,
+add the following configuration. Its value is 0(ms) by default, and certificate will not automatically refresh when its value <= 0.
```

Comment:
KeyVaultKeyStore can fetch certificates from KeyVault periodically if following property is configured:

```yaml
azure:
  keyvault:
    jca:
       certificates-refresh-interval: 1800000
```
Its value is 0(ms) by default, and certificate will not automatically refresh when its value <= 0.